samples/geography/requirements.txt

```diff
@@ -1,4 +1,48 @@
+attrs==21.2.0
+cachetools==4.2.2
+certifi==2021.5.30
+cffi==1.14.6
+charset-normalizer==2.0.4
+click==8.0.1
+click-plugins==1.1.1
+cligj==0.7.2
+dataclasses==0.6
```

Comment:
We should add the python version exclusion here while we're at it, since we know renovate is going to try to update this to 0.8 and it'll break.

Reply:
Aside: I know this is going to give us a lot of renovate PRs. We might want to cull this down, though I think there is some benefit of having a package list like this of all the dependencies and transitive dependencies we know work together.

Reply:
I added the Python version qualification for dataclasses.

IMO, it's best to pin the transitive dependencies.

Shall I merge after the tests pass?